members/views/unionOverview.py

```diff
@@ -0,0 +1,13 @@
+from django.shortcuts import render
+from django.contrib.auth.decorators import login_required, user_passes_test
+from members.models import Union
+from members.utils.user import has_user
+
+
+@login_required
+@user_passes_test(has_user, "/admin_signup/")
+def unionOverview(request):
+    # get unions
+    unions = Union.objects.all()
+
+    return render(request, "members/union_overview.html", {"unions": unions})
```

Comment:
Er der brug for login her? 

Det er lidt overkill med den kommentar, `Union.objects.all()` indikerer fint at man får fat i foreninger. Du kan også slå det sammen til en oneliner


```pyton
 def unionOverview(request):
      return render(request, "members/union_overview.html", {"unions": Union.objects.all()})
```

Reply:
Yeah det var bare en leftover fra tidligere udgaver, skal lige have den slettet.

Skulle ikke være behov for login, så vidt jeg har forstået kan alle se listen over foreninger og gå ind på alle foreningerne, det er kun til at se selve medlemslisten de skal være admins